HesoyamHospital/GraphicEditor/Model/EquipmentAndMedicine.cs

```diff
@@ -0,0 +1,28 @@
+﻿
+namespace GraphicEditor
```

Comment:
Is this a DTO object or a model?
If it's a model, it should be placed within a Backend project.
Contact Web team to check if they are already working on something similiar to reduce a chance for conflict and time loss.